### ISSUE TITLE
Multi-view links

### ISSUE THREAD

**Issue by druck13** [silvanmelchior/RPi_Cam_Web_Interface#511]
It would be very useful for each of the multi-view images to link to the web page for its camera.

**Comment by roberttidey** [comment]
Yes. I have thought about this in the past but not done anything on it yet.

It would be fairly straightforward to  to supply back links for each camera that multiview.html would add in. This could probably be done by adding in a second parameter for multiview.php to pass back.

One little complication is to provide a method for returning to multiview.
